### PR TITLE
Pludi feature ignore missing delete

### PR DESCRIPTION
**Pull request by MarkGibbons**
This change is [Reviewable](https://reviewable.io/reviews/sous-chefs/line-cookbook/83)

### CODE REVIEW

**Comment by MarkGibbons** [comment]
Merge the change from @pludi with some README updates.

**Comment by damacus** [comment]
I think we need to make a note that we do not create the file if it doesn't exist. Otherwise the user might think we go ahead and create the file. 

In light of this, this PR seems pretty wrong now. And we probably should raise an error, operating on a non-existant file, and ignoring it, the user will not get what they expect. 

- I want to delete a line in a file
- File doesn't exist
- Typically you'd expect even `rm` to error if it doesn't exist and you've explicitly asked it to delete a file, or `sed` will complain if the target doesn't exist.

**Comment by pludi** [comment]
> And we probably should raise an error, operating on a non-existant file, and ignoring it, the user will not get what they expect.

Which is why I didn't change the default behavior of raising an error if the file didn't exist, and requiring the option to be explicitly set to change this.

File: README.md [hunk 1]
```diff
@@ -88,11 +149,25 @@ multi = ([310], [818], [425])
...
+ends_with | List ending |  String | No default
+ignore_missing | Don't fail if the file is missing  |  true or false | Default is false
+### Notes
+Delimeters works exactly the same way as `add_to_list`, see above.
+
 
 # Author
-Author:: Sean OMeara (<sean@sean.io>)
-Contributor:: Antek S. Baranski (<antek.baranski@gmail.com>)
+Author: Sean OMeara (<sean@sean.io>)
```
Comment (damacus) [inline README.md:172]: Random additional white space? :/

**Comment by damacus** [comment]
@pludi that's excellent. I'm super happy with this feature then!

**Review by damacus — APPROVED** [review]

**Pull request merged**